src/app/data-planes/components/data-plane-traffic/DataPlaneTraffic.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="service-traffic">
+    <DataCard
+      class="header"
+    >
+      <template #title>
+        <slot name="title" />
+      </template>
+      <slot name="data" />
+    </DataCard>
+    <slot name="default" />
+  </div>
+</template>
+<script lang="ts" setup>
+import DataCard from '@/app/common/data-card/DataCard.vue'
+</script>
+<style lang="scss" scoped>
+.service-traffic {
+  display: flex;
+  flex-direction: column;
+  gap: $kui-space-40;
+}
+.kong-card {
+  &:is(.header) {
+    border: 0;
+  }
+}
```

Comment:
**Question**: There’s a lot of KCard overriding going on in the components here. We override `padding`, in this instance `border`, often `border-color` and `background-color`. Does it really make sense to use KCard here? It seems to me the cards are sufficiently different that trying to stick to using KCard turns out more of a hindrance than a benefit.

Reply:
Gonna resolve this one